src/riak_ql_parser.yrl

```diff
@@ -167,8 +169,8 @@ ColumnDefinition ->
 ColumnConstraint -> not_null : '$1'.
 
 DataType -> datetime : '$1'.
-DataType -> float : '$1'.
-DataType -> int : '$1'.
+DataType -> float_type : '$1'.
```

Comment:
These two lines still need to emit the type 'unchanged' because that will then feed into the DDL compiler

DataType -> float_type: float.
DataType -> int_type: int.

Which means the tests north of it will need to change